src/Controller/EventController.php

```diff
@@ -57,6 +61,9 @@ private function listenerMethods(string $eventClass): array|null
         );
```

Comment:
`$this->listenerProvider->listenersForEvent($eventClass)` return `iterable`, needs to be casted to `array` here